app/models/micropost.rb

```diff
@@ -1,6 +1,7 @@
 class Micropost < ApplicationRecord
   belongs_to :user
   scope :recent_posts, ->{order created_at: :desc}
+  scope :user_id, ->(id){where "user_id IN (?)", id}
```

Comment:
```suggestion
  scope :by_user_id, ->(ids){where "user_id IN (?)", ids}
```
thêm số nhìu thể heienj array nào